### PR TITLE
feat: adding tagging for live preview [TOL-909]

### PR DESCRIPTION
**Pull request by aodhagan-cf**
## Purpose of PR

- Adding tagging for live preview

### CODE REVIEW

**Review by YvesRijckaert — APPROVED** [review]

**Comment by aodhagan-cf** [comment]
@YvesRijckaert any ideas why there are no checks on this PR?

File: src/components/features/section-headlines/section-headlines.tsx [hunk 1]
```diff
@@ -33,8 +34,10 @@ const useStyles = makeStyles((theme: Theme) => ({
 interface SectionHeadlinesPropsInterface {
   headline?: string | null;
   headlineProps?: TypographyProps;
+  headlineLivePreviewProps?: ReturnType<typeof getLivePreviewProps>;
```
Comment (chrishelgert) [inline src/components/features/section-headlines/section-headlines.tsx:37]: Future Optimisation:
As this is probably a really common case, we could also provide the types from our SDK directly?

**Comment by chrishelgert** [comment]
> @YvesRijckaert any ideas why there are no checks on this PR?

As far as I know, you need to enable them manually on forked repositories:
https://github.com/contentful/template-marketing-webapp-nextjs-with-live-preview/actions

**Review by chrishelgert — APPROVED** [review]
Do we already have a demo space with this template? :)

File: src/components/features/section-headlines/section-headlines.tsx (hunk 1 above)
Comment (aodhagan-cf) [reply src/components/features/section-headlines/section-headlines.tsx:37]: will do in follow up

**Pull request merged**